### DISCUSSION TITLE
Creating Custom Image Keys With Links

### DISCUSSION THREAD

**Discussion by josephisticated** [maximmax42/Discord-CustomRP#110] [Ideas]
Discord won't let us upload gifs via the developer portal but allows the usage of third party image links, most of the time I just use links instead of uploading them to Discord Developer Portal as it is more convenient and most of the time I cant upload gifs to the developer portal. So I thought that it'd be nice if we could create image keys within the app, like, they'll be a button that says create a custom key and we insert a link and name it.

**Comment by maximmax42** [comment]
I mean, you can't upload gifs because presence doesn't support animated gifs afaik. As for uploading assets within the app, I think you can only upload through discord's website.

**Comment by josephisticated** [comment]
Rich presence DOES support gifs, just not through the developer portal but through third-party websites, you can check mine right now.

**Reply by maximmax42** [reply]
Well I'll see what I can do.
